python/adelphi/adelphi/gh.py

```diff
@@ -0,0 +1,34 @@
+# Functionality necessary to create a feature branch + pull request on Github
```

Comment:
We must avoid the name "github" for this module since it's use causes Python 2.x to have problems with the PyGithub import ("from github import ...") op that happens a few lines below.  This isn't a problem on Python 3.x, but to avoid the issue entirely I just renamed the module.